evals/scripts/get_total_fans_generator.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import string
+from get_context import get_context
+from get_instruction import get_instruction
+
+CURRENT_DIR = os.path.dirname(__file__)
+REGISTRY_PATH = os.path.join(CURRENT_DIR, "../registry")
+
+YAML = """
+get_total_unique_fans:
+  id: get_total_unique_fans.dev.v0
+  description: Evaluates the model's ability to analyze music streaming and fan engagement data, including metrics like premium vs free subscribers, playlist engagement, artist performance, and listener behavior across different streaming platforms.
+  metrics: [accuracy]
+get_total_unique_fans.dev.v0:
+  class: evals.elsuite.modelgraded.classify:ModelBasedClassify
+  args:
+    samples_jsonl: get_total_unique_fans/samples.jsonl
+    eval_type: cot_classify
+    modelgraded_spec: closedqa
+    modelgraded_spec_args:
+      criteria: "correctness: The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
+""".strip()
```

Comment:
Is this YAML the same in all script files?
- if yes, could you please DRY them?